Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig

```diff
@@ -11,7 +11,7 @@ file that was distributed with this source code.
 {% spaceless %}
 
     <input type="text" id="{{ id }}_autocomplete_input" value=""
-        {%- if read_only is defined and read_only %} readonly="readonly"{% endif -%}
+        {% for attribute,value in attr %} {{attribute}}="{{value}}" {% endfor %}
```

Comment:
I think you don't have to remove `{%-` and `-%}` they are there to avoid extra whitespaces